src/System.Collections.Specialized/src/System/Collections/Specialized/ListDictionary.cs

```diff
@@ -493,6 +492,7 @@ public void Reset()
         }
 
         [Serializable]
+        [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Maybe we were wrong and the `TypeForwardedFrom` attributes are needed even on nested types.